odonto/odonto_submissions/dpb_api.py

```diff
@@ -0,0 +1,45 @@
+import datetime
+import requests
+from django.conf import settings
+from requests.auth import HTTPBasicAuth
+from .exceptions import MessageSentException
+from . import logger
+
+
+CLAIMS_URL = "https://ebusiness.dpb.nhs.uk/claims.asp"
+RESPONSES_URL = "https://ebusiness.dpb.nhs.uk/responses.asp"
+
+
+def send_message(xml):
+    logger.info(
+        "message sent {} {}".format(
+            xml, datetime.datetime.now()
+        )
+    )
+    if settings.SEND_MESSAGES or not settings.DEBUG:
```

Comment:
It takes me ages to read what the logic of this actually is - compound statements on one line make my head hurt :D

Reply:
Actually the fact that you have both settings involved makes me think it could be better designed all round - you're trying to prevent hitting the url when the model saves in test, but also allow yourself to fiddle with another setting to override this setting behaviour ?